src/Game/gamePlayReport.cpp

```diff
@@ -9,6 +9,9 @@
 
 namespace uking {
 
+unsigned int getQuestId(const sead::SafeString& quest_name);
```

Comment:
can you remove this? I've implemented getQuestId (at line 58) so we don't need this declaration anymore.